core/src/reporter/data-feed-l2.ts

```diff
@@ -0,0 +1,24 @@
+import { Logger } from 'pino'
+import type { RedisClientType } from 'redis'
+import { factory } from './factory'
+import {
+  DATA_FEED_SERVICE_NAME,
+  DATA_FEED_REPORTER_L2_STATE_NAME,
+  REPORTER_AGGREGATOR_L2_QUEUE_NAME,
+  L2_PROVIDER_URL,
+  L2_CHAIN
+} from '../settings'
+
+export async function buildReporter(redisClient: RedisClientType, logger: Logger) {
+  await factory({
+    redisClient,
+    stateName: DATA_FEED_REPORTER_L2_STATE_NAME,
+    service: DATA_FEED_SERVICE_NAME,
+    reporterQueueName: REPORTER_AGGREGATOR_L2_QUEUE_NAME,
+    concurrency: 5,
+    delegatedFee: true,
```

Comment:
Do we plan to have a delegator on L2? If not, I think we should change the value to `false`.

Reply:
Yes and no. This gonna be a business decision if some L2 wants to integrate us. But it make more sense to keep it to false for now.